src/ResourceManager/Storage/Commands.Management.Storage/Commands.Management.Storage.csproj

```diff
@@ -135,12 +136,6 @@
     <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\..\..\Common\Commands.Common.Storage\Adapters\ARM.Storage.3\ARMStorageProvider.cs">
```

Comment:
@blueww I think the right thing here is to write the needed adapters for the new storage management client.  It looks like you copied over the methods instead.